flink-connectors/flink-jdbc/src/main/java/org/apache/flink/api/java/io/jdbc/JdbcBatchingOutputFormat.java

```diff
@@ -105,6 +119,66 @@ public void open(int taskNumber, int numTasks) throws IOException {
 		}
 	}
 
+	private void tryCreateTableIfNotExists(int taskNumber, JdbcDdlOptions jdbcDdlOptions) throws SQLException, ClassNotFoundException {
+		// only try to create table from one TM, simply choice the minimum TM as leader
+		// after FLIP-27，this can be refactor better.
+		final boolean isCreateTableLeader = taskNumber == 0 ? true : false;
+		final JDBCOptions jdbcOptions = jdbcDdlOptions.getJdbcOptions();
+		final int maxRetries = jdbcDdlOptions.getMaxRetry();
+
+		int retry = 1;
+		if (isCreateTableLeader) {
+			while (retry <= maxRetries) {
+				try (Connection conn = new SimpleJdbcConnectionProvider(jdbcOptions).getConnection(); Statement st = conn.createStatement()) {
```

Comment:
Reuse the `connection` initialized in `AbstractJdbcOutputFormat#establishConnection`?